packages/curve-ui-kit/src/widgets/Footer/Section.tsx

```diff
@@ -0,0 +1,32 @@
+import Grid from '@mui/material/Grid2'
+import Typography from '@mui/material/Typography'
+
+import { Link, LinkProps } from './Link'
+
+export type SectionProps = {
+  title: string
+  links: LinkProps[]
+}
+
+export const Section = ({ title, links }: SectionProps) => (
+  <Grid container spacing={1}>
+    <Grid size={12}>
+      <Typography
+        variant="headingXsBold"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          height: '2.5rem',
```

Comment:
```suggestion
          height: SizesAndSpaces.ButtonSize.sm,
```

Reply:
I'm using `SizesAndSpaces.ButtonSize.sm` for the height now in `Section.tsx`. In similar vain I'm using `height: SizesAndSpaces.ButtonSize.xs` instead of 1.5rem for `Link.tsx`.